install-dependencies.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+# This file is open source software, licensed to you under the terms
+# of the Apache License, Version 2.0 (the "License").  See the NOTICE file
+# distributed with this work for additional information regarding copyright
+# ownership.  You may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -euo pipefail
+
+. /etc/os-release
+
+fedora_packages=(
+    golang-bin
+    podman-docker
+)
+
+if [ "$ID" = "fedora" ]
```

Comment:
We need Ubuntu too.
Simillar script with support for more package types is here: https://github.com/scylladb/mermaid/blob/b4961ac16ea6e621d7497eeabaf23b70ecbc24a3/install_deps.sh#L1
I think we can clean it up a use it here.